frontend/app/styles/app.css

```diff
@@ -40,3 +40,7 @@ li a {
 #team-power-select {
   height: 69px;
 }
+
+.mb-10 {
```

Comment:
Give it a clearer name? margin-bottom-10?

Reply:
We already have margin which is named like this. So i made it like them, to maintain constancy.